editors/code/src/ctx.ts

```diff
@@ -4,21 +4,21 @@ import * as lc from 'vscode-languageclient';
 import { Config } from './config';
 import { createClient } from './client';
 import { isRustEditor, RustEditor } from './util';
-import { PersistentState } from './persistent_state';
 
 export class Ctx {
     private constructor(
         readonly config: Config,
-        readonly state: PersistentState,
+        // readonly state: PersistentState,
```

Comment:
If we won't use `getServer()` in the command handler, then we don't need this comment
```suggestion
```